test/e2e_node/docker_test.go

```diff
@@ -37,8 +39,37 @@ var _ = framework.KubeDescribe("Docker features [Feature:Docker]", func() {
 		framework.RunIfContainerRuntimeIs("docker")
 	})
 
-	Context("when shared PID namespace is enabled", func() {
-		It("processes in different containers of the same pod should be able to see each other", func() {
+	Context("when pod PID namespace is configurable", func() {
```

Comment:
We should move the test out of the docker suite and add a proper feature tag (e.g., `Feature:ShareProcessNamespace`). WDYT?

Reply:
Good point, this isn't docker-specific anymore. I've moved it into the security_context tests alongside the other namespace tests. PTAL.